### PR TITLE
Industrial Expansion

### PR DESCRIPTION
**Pull request by Mechoid**
:cl:
add - Adds more materials from World Server. The progress toward Rimworld-but-multiplayer continues.
add - Adds copper to multiple items' costs and default matter values. Especially important: Most circuits.
tweak - Autolathes, Protolathes, Circuit Printers, and Mechfabs can take any material. Most are hidden unless inserted.
add - Autolathes can produce 14.5 mm sabot shells.
add - Protolathes can produce 14.5 mm shells.
tweak - Autolathes can now hold equivalent amounts of Glass and Steel, and any other material, for ease of operation with procedurally acquired storage.
maptweak - Copper added to Engineering storage and Miscellaneous Research.
tweak - Various R&D designs updated to make use of certain new materials.
/:cl:

### CODE REVIEW

**Comment by Schnayy** [comment]
What do these new materials contribute to the game? What gameplay aspect is improved by having more niche materials to manage for RND? I'm not sold on this.

**Comment by Mechoid** [comment]
Copper is utilized for electronics, Titanium and Aluminum take the place of Steel & Plasteel in some designs as Steel is not as light as either.

File: code/datums/autolathe/autolathe.dm [hunk 1]
```diff
@@ -2,7 +2,14 @@ var/datum/category_collection/autolathe/autolathe_recipes
 
 /datum/category_item/autolathe/New()
 	..()
-	var/obj/item/I = new path()
+	var/obj/item/I
+	if(path)
+		I = new path()
+
+	if(!I)	// Something has gone horribly wrong, or right.
+		log_debug("[name] created an Autolathe design without an assigned path. This is expected for only the Material Sheet generation.")
```
Comment (Atermonera) [inline code/datums/autolathe/autolathe.dm:10]: If it's expected for something then that something should suppress the log

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -53,7 +53,14 @@ var/list/mining_overlay_cache = list()
...
 		"verdantium" = /obj/item/weapon/ore/verdantium,
 		"marble" = /obj/item/weapon/ore/marble,
-		"lead" = /obj/item/weapon/ore/lead
+		"lead" = /obj/item/weapon/ore/lead,
+		"copper" = /obj/item/weapon/ore/copper,
+		"tin" = /obj/item/weapon/ore/tin,
+		"bauxite" = /obj/item/weapon/ore/bauxite,
+//		"void opal" = /obj/item/weapon/ore/void_opal,
+//		"painite" = /obj/item/weapon/ore/painite,
+//		"quartz" = /obj/item/weapon/ore/quartz,
```
Comment (Atermonera) [inline code/modules/mining/mine_turfs.dm:62]: ?

Comment (Mechoid) [reply code/modules/mining/mine_turfs.dm:62]: I have plans for Void Opal, Painite, and Quartz in future PRs, and I didn't quite want to re-find where I put all the additions the first time. Mining changes are tedious as all hell

File: code/datums/autolathe/autolathe.dm (hunk 1 above)
Comment (Mechoid) [reply code/datums/autolathe/autolathe.dm:10]: These only fire off for Materials once per type, and wherein they actually have sheets. I think it's safer to have it remain for them incase something explodes later on for autolathes, in the manner of a canary.

**Comment by chizzy376** [comment]
But what is the point of adding new materials to R&D? We really do not need them..

**Pull request merged**